rdt/transformers/base.py

```diff
@@ -73,13 +74,16 @@ def get_supported_sdtypes(cls):
         """
         return cls.SUPPORTED_SDTYPES or [cls.INPUT_SDTYPE]
 
-    def _add_prefix(self, dictionary):
-        if not dictionary:
-            return {}
-
+    def _get_output_to_property(self, property_):
         output = {}
-        for output_columns, output_sdtype in dictionary.items():
-            output[f'{self.column_prefix}.{output_columns}'] = output_sdtype
+        for output_column, properties in self.output_properties.items():
+            # if 'sdtype' is not in the dict, ignore the column
+            if property_ not in properties:
```

Comment:
**Note**: this is required for the special case where transformers don't pass an 'sdtype'. In such cases we don't want to add the output_column at all (instead of adding the `column_prefix: None` pair to the dict) because `_build_output_columns` adds a # to the end of generated column which already exit in the data.

Reply:
what happens if `property_` is set to `'next_transformer'`? Do we still want to skip if the dict doesn't contain it?

Reply:
That never happens (and should never happen). If you mess up and forget to add a `next_transformer` to `output_properties`, skipping it is the best we can do.